functions/zoom-downloader.py

```diff
@@ -119,17 +127,47 @@ def zoom_series_id(self):
     def start_time(self):
         return self.data['start_time']
 
+    @property
+    def end_time(self):
+        return self.data['end_time']
+
+    @property
+    def duration(self):
+        return self.data['duration']
+
     @property
     def created(self):
         utc = datetime.strptime(
-            self.start_time, '%Y-%m-%dT%H:%M:%SZ') \
+            self.start_time, TIMESTAMP_FORMAT) \
             .replace(tzinfo=timezone('UTC'))
         return utc
 
+    @property
+    def shorter_than_minimum_duration(self):
+        """
+        Interpret duration of meeting and determine whether duration is
+        shorter than the MINIMUM_DURATION in minutes.
+        """
+        if self.duration:
+            duration_in_mins = time.strptime(self.duration, "%H:%M:%S").tm_min
+        elif self.start_time and self.end_time:
+            s = datetime.strptime(self.start_time, TIMESTAMP_FORMAT)
+            e = datetime.strptime(self.end_time, TIMESTAMP_FORMAT)
+            delta = e - s
+            duration_in_mins = delta.seconds // 60
+        else:
+            # Default behavior
+            # do not filter out recording if duration cannot be found
+            duration_in_mins = MINIMUM_DURATION
+
+        return duration_in_mins < MINIMUM_DURATION
+
```

Comment:
This was the original purpose of this branch. Filter out extremely short videos since we've had 2 tickets about sub-1-minute videos getting through to opencast this semester.